#codes incremental change for goal postion in x,y,z-direction
int64 button_code # 1 x-, 2 x+, 3 y-, 4 y+, 5 z-, 6 z+
---

